src/lambda/configEditor/vue/samInvokeFrontend.ts

```diff
@@ -152,6 +158,7 @@ export default defineComponent({
             if (!config) {
                 return
             }
+            const company = this.company
             this.clearForm()
```

Comment:
xxx: save `company` so it can be restored after clearForm()

Reply:
Yeah  `getCompanyName` would be called again. This particular webview doesn't get stored in memory when not visible. If the user unfocuses the tab (or refreshes), `created` would be called again despite potentially being the same iframe. The `saveData` mixin is what persists `data` through entire reloads.